src/effects/RealtimeEffectStateUI.cpp

```diff
@@ -156,3 +166,7 @@ void RealtimeEffectStateUI::UpdateTitle()
    mEffectUIHost->SetName(title);
 }
 
+void RealtimeEffectStateUI::AutoSave(AudacityProject &project)
+{
+   ProjectHistory::AutoSave::Call(project);
```

Comment:
I still observe this call during the playback. I had an impression that this is wrong?

Reply:
Playback?  Which steps to reproduce are you referring to?

Reply:
Hmm, looks like I've missed something. So this change is really about moving ProjectWindowDestroyedMessage?

Reply:
Updated steps to reproduce at https://github.com/audacity/audacity/issues/3663 explain that playback was not a necessary part